convert_util.c

```diff
@@ -131,6 +132,27 @@ convert_parse_tlvs(const uint8_t *buff, size_t buff_len)
 
 			break;
 		}
+		case CONVERT_EXTENDED_TCP_HDR: {
+			struct convert_extended_tcp_hdr *conv_ext_tcp_hdr =
+			        (struct convert_extended_tcp_hdr *)buff;
+			size_t tcp_options_len = tlv_len -
```

Comment:
maybe go to the next line after the `= `

Reply:
Yes will try that and see if Uncrustify agrees :-)